utils/connectToStores.js

```diff
@@ -0,0 +1,94 @@
+'use strict'
+import React from 'react'
```

Comment:
09b64ec removes React from the build, but not sure if `window.React` gets picked up. Need to work up a client-side test I guess.